src/Fixture/AbstractAssetFixture.php

```diff
@@ -0,0 +1,39 @@
+<?php declare(strict_types=1);
+
+namespace Neusta\Pimcore\FixtureBundle\Fixture;
+
+use Pimcore\Model\Asset;
+use Pimcore\Model\Asset\Image;
+
+abstract class AbstractAssetFixture extends AbstractFixture
+{
+    public function __construct(
+        protected string $fullQualifiedFilename,
+        protected string $pimcoreFilename,
+        protected string $pimcoreBasePath,
+        protected string $fixtureAssetMarker,
```

Comment:
I don't really like those parameter names...

I propose to change them (especially if we do https://github.com/teamneusta/pimcore-fixture-bundle/pull/30#discussion_r1973241270) like this:

```suggestion
        protected string $fileContent,
        protected string $filename,
        protected string $basePath,
        protected string $reference,
```